.github/workflows/documentation.yml

```diff
@@ -0,0 +1,33 @@
+name: Documentation
```

Comment:
This'll be the badge name as well.  Do we want to shorten it to something like `docs` (lowercase) to match the other badge styles?